style/battle.css

```
@@ -789,6 +789,9 @@ License: GPLv2
 #tooltipwrapper .tooltip p.section {
 	border-top: 1px solid #888888;
 }
+#tooltipwrapper .tooltip span.textaligned-typeicons {
+	vertical-align: -3px;
+}
```

Rather than hard-coding an adjustment on the wrapper, wouldn't it be better to add a class to the type icons themselves so that you could style them with `vertical-align: text-bottom;` or `middle` (not sure which one would be better)?

---

Although the type icon in the status indicator gets away without using a class on the icon itself, so maybe just leave the span in and style the icon using
```suggestion
#tooltipwrapper .tooltip .textaligned-typeicons img {
	vertical-align: text-bottom;
}
```